package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "3.53.1",
+  "version": "3.53.2",
```

Comment:
Do we need to bump this now when we modify one of our components? I didn't know.

Reply:
I assumed do.  It's probably sensible to increment the minor version as it's a different codebase but not a change in functionality.

Reply:
We only bump the version and update the CHANGELOG of the root when we modify something at the root level. If we only modify components, we bump and update the CHANGELOGs of said components. So in this case I think we need to revert this change.